### PR TITLE
add mkdocs-jupyter + markdown-include + mkapi + pdoc

### PR DESCRIPTION
**Pull request by hadim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hadim** [comment]
Generated with grayskull.

**Comment by hadim** [comment]
@conda-forge/help-python please review

**Comment by hadim** [comment]
I have also included markdown-include.

**Comment by hadim** [comment]
mkapi added too.

**Comment by hadim** [comment]
@conda-forge/help-python please review

**Comment by hadim** [comment]
@conda-forge/help-python please review

File: recipes/markdown-include/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - markdown_include
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/cmacmackin/markdown-include/
+  summary: This is an extension to Python-Markdown which provides an "include" function, similar to that found in LaTeX (and also the C pre-processor and Fortran). I originally wrote it for my FORD Fortran auto-documentation generator.
+  license: GPL-3.0
```
Comment (chrisburr) [inline recipes/markdown-include/meta.yaml:37]:
```suggestion
  license: GPL-3.0-or-later
```

File: recipes/markdown-include/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  imports:
+    - markdown_include
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/cmacmackin/markdown-include/
+  summary: This is an extension to Python-Markdown which provides an "include" function, similar to that found in LaTeX (and also the C pre-processor and Fortran). I originally wrote it for my FORD Fortran auto-documentation generator.
```
Comment (chrisburr) [inline recipes/markdown-include/meta.yaml:36]:
```suggestion
  summary:  Provides syntax for Python-Markdown which allows for the inclusion of the contents of other Markdown documents. 
  description: |
  This is an extension to Python-Markdown which provides an "include" function,
  similar to that found in LaTeX (and also the C pre-processor and Fortran). I
  originally wrote it for my FORD Fortran auto-documentation generator.
```

File: recipes/markdown-include/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (chrisburr) [inline recipes/markdown-include/meta.yaml:21]:
```suggestion
    - python >=2.7
```

File: recipes/markdown-include/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  run:
+    - markdown
+    - python
```
Comment (chrisburr) [inline recipes/markdown-include/meta.yaml:24]:
```suggestion
    - python >=2.7
```

**Comment by hadim** [comment]
Thank you @chrisburr

**Comment by hadim** [comment]
LGTM

**Pull request merged**